docs/resources/cci_namespace.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Cloud Container Instance (CCI)"
+---
+
+# huaweicloud_cci_namespace
+
+Manages a CCI namespace resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "namespace_name" {}
+
+resource "huaweicloud_cci_namespace" "test" {
+  name               = var.namespace_name
+  flavor             = "gpu-accelerated"
+  pool_size          = 15
+  recycling_interval = 30
+  auto_expend        = true
+  container_network  = true
+  rbac_enable        = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the CCI namespace resource.
+  If omitted, the provider-level region will be used. Changing this will create a new CCI namespace resource.
+
+* `flavor` - (Required, String, ForceNew) Specifies the CCI namespace flavor.
+  The valid values are **general-computing** and **gpu-accelerated**.
+  Changing this will create a new CCI namespace resource.
+
+* `name` - (Required, String, ForceNew) Specifies the unique name of the CCI namespace.
+  This parameter can contain a maximum of 63 characters, which may consist of lowercase letters, digits and hyphens,
+  and must start and end with lowercase letters and digits.
+  Changing this will create a new CCI namespace resource.
+
+* `auto_expend` - (Optional, Bool, ForceNew) Specifies whether elastic scheduling is enabled.
+  Changing this will create a new CCI namespace resource.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies a unique ID in UUID format of enterprise project.
+  Changing this will create a new CCI namespace resource.
+
+* `pool_size` - (Optional, Int, ForceNew) Specifies the size of IP pool to warm-up.
+  Changing this will create a new CCI namespace resource.
+
+* `recycling_interval` - (Optional, Int, ForceNew) Specifies the IP address recycling interval, in hour.
+  Changing this will create a new CCI namespace resource.
+
+* `container_network` - (Optional, Bool, ForceNew) Specifies whether container network is enabled.
```

Comment:
Enable this option if you want CCI to start the container network in advance so that containers can connect to the network as soon as they are started.

Reply:
The description has been added.